src/Microsoft.AspNetCore.Http.Connections/ConnectionsDependencyInjectionExtensions.cs

```diff
@@ -6,8 +6,16 @@
 
 namespace Microsoft.Extensions.DependencyInjection
 {
+    /// <summary>
+    /// Extension methods for <see cref="IServiceCollection"/>.
+    /// </summary>
     public static class ConnectionsDependencyInjectionExtensions
     {
+        /// <summary>
+        /// Adds HTTP connection functionality to the specified <see cref="IServiceCollection" />.
```

Comment:
See above comment on `UseConnections`. Something like "Adds required services for ASP.NET Core Connection Handlers to the specified [IServiceCollection]"